tools/static-code-analysis/checkstyle/suppressions.xml

```diff
@@ -20,4 +20,5 @@
     <suppress files=".+org.eclipse.smarthome.automation.+" checks="HeaderCheck"/>
     <suppress files=".+org.eclipse.smarthome.config.dispatch.test.+" checks="ServiceComponentManifestCheck"/>
     <suppress files=".+org.eclipse.smarthome.core.thing.+" checks="PackageExportsNameCheck"/>
+    <suppress files=".+org.eclipse.smarthome.automation.+" checks="ServiceComponentManifestCheck"/>
```

Comment:
Why do we need this suppression?

Reply:
The plugin that processes the DS annotations adds a Service-Component manifest header, which is not allowed by the the code style checker. The build fails without this suppression.

Reply:
Could you check why it flawlessly works on all our other bundles? There should be no different situation here.

Reply:
I see the same suppression here `<suppress files=".+org.eclipse.smarthome.config.dispatch.test.+" checks="ServiceComponentManifestCheck"/>`, but I'll try to find another way to solve this problem

Reply:
This is merely for a test bundle - for all "real" bundles, we do not have such suppressions.